documentation/docs/30-advanced/10-advanced-routing.md

```diff
@@ -123,27 +123,33 @@ src/routes/[...catchall]/+page.svelte
 
 ### Encoding
 
-Directory names are URI-decoded, meaning that (for example) a directory like `%40[username]` would match characters beginning with `@`:
+Some characters can't be used on the filesystem — `/` on Linux and Mac, `\ / : * ? " < > |` on Windows. The `#` character has special meaning in URLs, and the `[ ] ( )` characters have special meaning to SvelteKit, so these also can't be used directly as part of your route.
 
-```js
-// @filename: ambient.d.ts
-declare global {
-	const assert: {
-		equal: (a: any, b: any) => boolean;
-	};
-}
+To use these characters in your routes, you can use hexadecimal escape sequences:
```

Comment:
I would add a small note here that the format is `[x/u+code]`, and after the list possibly link to some code table where you can look up more of them

Reply:
added some docs